docs/make.jl

```diff
@@ -18,7 +18,9 @@ makedocs(
                   "nemo.md"
              ],
             "polynomial.md",
-            "ideal.md",
+            "weyl.md",
+	    "exterior.md",
+	    "ideal.md",
```

Comment:
Your editor probably has some weird settings:
1. Make sure that tabs are replaced by spaces so that these line up for everyone who looks at your code.
2. Make sure that characters over 127 are encoded properly (julia seems to want UTF-8), or just avoid them all together.

Reply:
I would make these changes, but the editor is broken and has been for years. It took me half an hour to figure out how to get out of "zen" mode the other day. If I'd known that was what it was called, I could have done it. Unfortunately it is the best editor available for this platform.

Reply:
The Groebner changes were made by me because what was there was not displayable on Windows. I guess we have to go for Groebner.

Reply:
I fixed the Groebner and spacing issues.